packages/framer-motion/src/components/AnimatePresence/index.tsx

```diff
@@ -75,13 +75,16 @@ function onlyElements(children: ReactNode): ReactElement<any>[] {
  *
  * @public
  */
-export const AnimatePresence: React.FunctionComponent<React.PropsWithChildren<AnimatePresenceProps>> = ({
+export const AnimatePresence: React.FunctionComponent<
+    React.PropsWithChildren<AnimatePresenceProps>
+> = ({
     children,
     custom,
     initial = true,
     onExitComplete,
     exitBeforeEnter,
     presenceAffectsLayout = true,
+    popLayout = false,
```

Comment:
I don't see the `custom` prop being used anywhere, but I was thinking, couldn't we make something like:
`transition={"pop"}`, so we allow for flexibility into the future?

Reply:
In theory I would be open to this but it seems to me like a binary behaviour - either elements are popped from layout while exiting or they're not. I can't imagine the "category" of behaviour this lives within, ie its not a transition or animation, nor how it would be incompatible with those other behaviours as a boolean.

Reply:
I misunderstood what the prop was doing, it seems fine like a boolean, but is there a better name than `pop` for it? I can't really come up with it, but might be worth trying a bit of other options.